src/common/utils/language-utils.js

```diff
@@ -13,9 +13,31 @@ const LANGUAGES = [
   'ko',
 ];
 
+const TAG_LANGUAGES = {
+  en: 'en',
+  de: 'de',
+  es: 'es',
+  fr: 'fr',
+  it: 'it',
+  ja: 'ja',
+  nl: 'nl',
+  'pt-br': 'pt',
+  sv: 'sv',
+  'zh-hant': 'tw',
+  'zh-hans': 'zh',
+  ko: 'ko',
+};
+
 export const getMatchLanguage = (lang) => {
   if (lang) {
     return LANGUAGES.find((l) => l.toLowerCase() === lang.toLowerCase());
   }
   return lang;
 };
+
+export const getMatchLanguageForTag = (lang) => {
+  if (lang) {
+    return TAG_LANGUAGES[lang] || 'default';
+  }
+  return 'default';
+};
```

Comment:
We can simplify to

`export const getMatchLanguageForTag = (lang) => TAG_LANGUAGES[lang] || 'default';`

however, we need to set it as 'en' right instead of 'default'

Reply:
Done